spinalcordtoolbox/labels.py

```diff
@@ -370,7 +370,8 @@ def continuous_vertebral_levels(img: Image) -> Image:
     #   a. extract centerline
     #   b. for each slice, extract corresponding level
     nx, ny, nz, nt, px, py, pz, pt = img.dim
-    _, arr_ctl, _, _ = get_centerline(img, param=ParamCenterline())
+    _, centerline, _ = get_centerline(img, param=ParamCenterline())
+    arr_ctl = centerline.arr_ctl
```

Comment:
arr_ctl is not used anywhere else. It is possible to extract directly the x, y and z components?

Reply:
Definitely! 

This was done for the same reasoning as https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4010#discussion_r1106271333, and there is definitely cleanup that can be done here.

Reply:
This concern has been addressed, because the highlighted changes have been reverted in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4010#discussion_r1108902353.